app/uk/gov/hmrc/customs/notification/services/NotificationDispatcher.scala

```diff
@@ -28,3 +35,26 @@ trait NotificationDispatcher {
   def process(csids: Set[ClientSubscriptionId])(implicit hc: HeaderCarrier): Future[Unit]
 
 }
+
+@Singleton
+class NotificationDispatcherImpl @Inject()(lockRepo: LockRepo, logger: NotificationLogger) extends NotificationDispatcher {
+
+  private val duration = Duration.standardMinutes(2) //TODO MC this should be configurable property (Avinder will make this change)
+
+  override def process(csids: Set[ClientSubscriptionId])(implicit hc: HeaderCarrier): Future[Unit] = {
```

Comment:
don't ask for header carrier, this method will be called from db poller and notification service.